src/reff.js

```diff
@@ -64,14 +64,23 @@ export function reff(output, rt, beta, population, mixingMatrix) {
     throw Error("mismatch between population and mixing matrix size");
   }
 
-  const tNow = rt.length;
+  if (beta.length !== rt.length) {
+    throw Error("mismatch between rt and beta size");
+  }
+
+  if (tSubset == null) {
```

Comment:
should this be `===`?

Reply:
It could be! I'm allowing for tSubset to be `undefined` here.

So the user could write something hideous like:

```js
const pars_1 = {
  beta: ...,
  population: ...,
  mixingMatrix: ...
};

const pars_2 = {
  beta: ...,
  population: ...,
  mixingMatrix: ...,
  subset: ...
};

for (let pars of [pars_1, pars_2]) {
  //when subset is not set we get default behaviour
  reff(output, rt, pars.beta, pars.population, pars.mixingMatrix, pars.subset);
}
```